app/services/auth.js

```diff
@@ -161,7 +161,7 @@ export default Service.extend({
     if (proVersion && get(config, 'intercom.enabled')) {
       this.get('intercom').set('user.name', user.name);
       this.get('intercom').set('user.email', user.email);
-      // FIXME createdAt?
+      this.get('intercom').set('user.createdAt', user.last_synced_at);
```

Comment:
ohhhhh, no no no, `first_logged_in_at`

Reply:
oops thanks, I didn’t get enough sleep 😳

Reply:
I finally got it running locally, looks like it’s `synced_at`:

![image](https://user-images.githubusercontent.com/43280/38696753-b5c2b9b4-3e55-11e8-91d5-02944385cfa4.png)

Reply:
er wait… `first_logged_in_at` doesn’t exist, or anything like it

Reply:
(for observers: `created_at`, obvsy; like I said, underslept 😆)